source/agora/api/Registry.d

```diff
@@ -17,6 +17,7 @@ import agora.common.Types;
 import agora.crypto.Hash;
 import agora.crypto.Key;
 import agora.crypto.Schnorr: Signature;
+import agora.flash.Node : KnownChannel;
```

Comment:
This means that the interface imports the implementation, which is a red flag.
If `KnownChannel` is part of the API, then it should be isolated (in another module) from the implementation.
The problem is that `KnownChannel` also contains `ChannelConfig`. Looking at the type, the `Config` module stands out a bit. It mixes the actual configuration file definitions (mirroring `agora.node.Config`) and Channel configuration, which, although it might be present in the configuration as well, is more importantly part of the interface.

Reply:
I moved all that to Flash api module, in a separate commit.